service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -106,6 +108,17 @@ private void addOrganization(Bundle bundle, Organization organization) {
         }
     }
 
+    private void addIncomingReferral(Bundle bundle, Encounter encounter) {
+        ReferralRequest referralRequest = (ReferralRequest) encounter.getIncomingReferralFirstRep().getResource();
+        addEntry(bundle, referralRequest);
+        if(referralRequest.hasSubject()){
+            addEntry(bundle, referralRequest.getSubjectTarget());
+        }
+        if(referralRequest.hasContext()){
```

Comment:
No need to add context to the bundle - it's the encounter - it's already in the bundle.